### PR TITLE
Add operations processes to team homepage

### PR DESCRIPTION
**Pull request by jplahn**

### CODE REVIEW

File: content/departments/engineering/teams/repo-management/index.md [hunk 1]
```diff
@@ -33,6 +33,59 @@ Including maintenance, development, support, and advice for:
...
+
+The centerpiece of our operations is the oncall. The oncall is our go-to person for handling inbound support requests, whether those are automated alerts, customer questions, or incidents. Oncall ownership rotates every week. We also utilize a secondary oncall to provide follow-the-sun oncall coverage for high severity issues and to load balance if the primary oncall has too many issues. Our oncall allocates their time to items in the following priority order:
+
+1. Pageable alerts
+2. Active incidents
+3. Customer p0s and p1s
+4. Sentry alerts
+5. Customer p2s
+6. Monitor / action / resolve errors in #repo-management-errors channel on Slack
+7. Questions in #repo-management and #ask-engineering
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/repo-management/index.md:50]: Let's add links to channels
```suggestion
7. Questions in [#repo-management](https://sourcegraph.slack.com/archives/C02EDAQAJQZ) and [#ask-engineering](https://sourcegraph.slack.com/archives/C022SPMNR0W)
```

File: content/departments/engineering/teams/repo-management/index.md [hunk 2]
```diff
@@ -33,6 +33,59 @@ Including maintenance, development, support, and advice for:
...
+### Oncall
+
+The centerpiece of our operations is the oncall. The oncall is our go-to person for handling inbound support requests, whether those are automated alerts, customer questions, or incidents. Oncall ownership rotates every week. We also utilize a secondary oncall to provide follow-the-sun oncall coverage for high severity issues and to load balance if the primary oncall has too many issues. Our oncall allocates their time to items in the following priority order:
+
+1. Pageable alerts
+2. Active incidents
+3. Customer p0s and p1s
+4. Sentry alerts
+5. Customer p2s
+6. Monitor / action / resolve errors in #repo-management-errors channel on Slack
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/repo-management/index.md:49]:
```suggestion
6. Monitor / action / resolve errors in [#repo-management-errors](https://sourcegraph.slack.com/archives/C045RUZAZ9P) channel on Slack
```

File: content/departments/engineering/teams/repo-management/index.md [hunk 3]
```diff
@@ -33,6 +33,59 @@ Including maintenance, development, support, and advice for:
...
+7. Questions in #repo-management and #ask-engineering
+8. Incident action items (if not prioritized during planning)
+9. Tech debt
+
+We have the following expectations of the oncall:
+
+1. Triaging and resolving support requests is your only priority for the week. Any and all project work is put on hold for the week. When in doubt, reference the priorities above.
+2. Monitor #repo-management and #ask-engineering for inbound requests.
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/repo-management/index.md:57]:
```suggestion
2. Monitor [#repo-management](https://sourcegraph.slack.com/archives/C02EDAQAJQZ) and [#ask-engineering](https://sourcegraph.slack.com/archives/C022SPMNR0W) for inbound requests.
```

File: content/departments/engineering/teams/repo-management/index.md [hunk 4]
```diff
@@ -33,6 +33,59 @@ Including maintenance, development, support, and advice for:
...
+3. Monitor any code monitors that post to our Slack channels
+4. Aim to acknowledge all requests within 24 hours, even if the initial response is an indication that we don’t have bandwidth to review it yet and will respond back at a future date. Add the GitHub issue to the Repo Management project with the Status “Support Issues”.
+5. Provide the incoming support owner with the status of all in-flight issues before their rotation begins (Monday morning is fine, asynchronous updates are fine). Include the relative priority of all issues so the incoming support owner knows where to begin.
+6. Ask for help when you are stuck! Don’t spend too much time trying to troubleshoot an issue, especially if it’s high priority. Do your best, ask the team, and resolve the issue.
+7. If you ever have too many support requests, leave a message in #repo-management-internal and engage the secondary support owner for the week
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/repo-management/index.md:62]:
```suggestion
7. If you ever have too many support requests, leave a message in [#repo-management-internal](https://sourcegraph.slack.com/archives/C02DUJRQP70) and engage the secondary support owner for the week
```

File: content/departments/engineering/teams/repo-management/index.md [hunk 5]
```diff
@@ -33,6 +33,59 @@ Including maintenance, development, support, and advice for:
...
+
+1. 2 or more repos have not synced for more than 9 hours
+
+The following are not pageable issues:
+
+1. Partial or delayed repository sync that has occurred for less than 9 hours
+2. Repo syncing issues that occur during the 7 day instance creation window
+3. 1 repository is not syncing
+
+We ask that you send us a message in #repo-management for not pageable issues so we can help you.
```
Comment (sashaostrikov) [inline content/departments/engineering/teams/repo-management/index.md:83]:
```suggestion
We ask that you send us a message in [#repo-management](https://sourcegraph.slack.com/archives/C02EDAQAJQZ) for not pageable issues so we can help you.
```

**Review by sashaostrikov — APPROVED** [review]
Thanks, everything LGTM!
I've added suggestions to add links to Slack channels

**Review by ryanslade — APPROVED** [review]

File: content/departments/engineering/teams/repo-management/index.md (hunk 5 above)
Comment (jplahn) [reply content/departments/engineering/teams/repo-management/index.md:83]: The handbook is magic and will automatically link to the Slack channel 🙂

Comment (sashaostrikov) [reply content/departments/engineering/teams/repo-management/index.md:83]: 🤯

**Pull request merged**